src/postalcode.js

```diff
@@ -0,0 +1,72 @@
+define([
+	"cldr",
+	"./core",
+	"./common/validate/cldr",
+	"./common/validate/default-locale",
+	"./common/validate/parameter-presence",
+	"./common/validate/parameter-type/string",
+
+	"cldr/event",
+	"cldr/supplemental"
+], function( Cldr, Globalize, validateCldr, validateDefaultLocale,
+	validateParameterPresence, validateParameterTypeString ) {
+
+/**
+ * .validatePostalCode( value )
+ *
+ * @value [String]
+ *
+ * Returns true if the value matches the postalcode pattern for the given locale,
+ * otherwise returns false
+ *
+ * Return the boolean.
+ */
+Globalize.validatePostalCode =
+Globalize.prototype.validatePostalCode = function( value, flag ) {
+	validateParameterPresence( value, "value" );
+	validateParameterTypeString( value, "value" );
+	return this.postalCodeValidator()( value, flag );
+};
+
+Globalize.postalCodeValidator =
+/**
+ * .postalCodeValidator()
+ *
+ * Return a validatePostalCode function (of the form below).
+ *
+ * fn( value )
+ *
+ * @value [String]
+ *
+ * Returns true if the value matches the postalcode pattern for the given locale,
+ * otherwise returns false
+ *
+ * Return the boolean.
+ *
+ */
+	Globalize.prototype.postalCodeValidator = function() {
+		var cldr, regex, pattern;
+
+	cldr = this.cldr;
+
+	validateDefaultLocale( cldr );
+
+	cldr.once( "get", validateCldr );
+	pattern = "^" + cldr.supplemental("postalCodeData/{region}") + "$";
```

Comment:
@rxaviers 
Don't we need to check validation of "region" ? like "validateParameterTypeString(region, "region") "?